src/apps/com.ximalaya.ting.android.ts

```diff
@@ -221,15 +221,20 @@ export default defineGkdApp({
       name: '权限提示-通知权限',
       desc: '取消推送通知',
       fastQuery: true,
-      matchTime: 10000,
       actionMaximum: 1,
       resetMatch: 'app',
       rules: [
         {
-          matches: '[id="com.ximalaya.ting.android:id/cancel_btn"]',
-          exampleUrls:
+          matches:
+            '@[id="com.ximalaya.ting.android:id/cancel_btn"] + [id="com.ximalaya.ting.android:id/btn_separator_border_1"] + [id="com.ximalaya.ting.android:id/ok_btn"]',
```

Comment:
简单 +2 即可，因为这里不需要关注中间用于分隔的节点 ID
```suggestion
            '@[id="com.ximalaya.ting.android:id/cancel_btn"] +2 [id="com.ximalaya.ting.android:id/ok_btn"]',
```

Reply:
当然如果是我写的话，我可能还会联系一下节点 `text="打开推送通知"`，看看 @AIsouler 的意见。